openforcefield/topology/molecule.py

```diff
@@ -3176,6 +3306,222 @@ def from_openeye(oemol, allow_undefined_stereo=False):
         return toolkit.from_openeye(
             oemol, allow_undefined_stereo=allow_undefined_stereo)
 
+    def to_qcshema(self, multiplicity=1, conformer=0):
```

Comment:
@j-wags

Reply:
We might consider changing this function to be part of a new QCElemental ToolkitWrapper in a later PR. That way, we could do the full molecule validation _in this function_, and return a QCElemental mol afterwards (instead of returning an un-validated dict, as we do now)

Reply:
I think that would be a safer way to do it, shall we look at making a wrapper for this part now? or should I just try and import QCElemental and throw an error if not?

Reply:
I'm redefining some conventions in `ToolkitWrapper` right now, so let's not make a `QCElementalToolkitWrapper` just yet. For now let's keep the code in `molecule.py`. Later, we can make the QCEl ToolkitWrapper, migrate this code to there, and change this function to just do `toolkit_registry.call('to_qcschema', mol, ...)`